src/array_api_extra/_lib/_testing.py

```diff
@@ -81,23 +83,28 @@ def _check_ns_shape_dtype(
     return desired_xp
 
 
-def _prepare_for_test(array: Array, xp: ModuleType) -> Array:
```

Comment:
Just to be clear, I didn't modify the signature of this function. This was what it was before the PR, right?